TrafficPPV2/Initializer.cs

```diff
@@ -381,15 +381,20 @@ void ReplacePathManager()
         //    Logger.LogInfo("Transport Manager successfully replaced.");
         //}
 
-        T TryGetComponent<T>(string name)
+        T TryGetComponent<T>(string name) where T: MonoBehaviour
         {
             foreach (string prefix in sm_collectionPrefixes)
             {
-                GameObject go = GameObject.Find(prefix + name);
-                if (go != null)
-                    return go.GetComponent<T>();
```

Comment:
This may be false when European Buildings Unlocker is enabled because it loads multiple game objects of the same name, but only some of them will have collections attached. If you have some similar lookups elsewhere you should fix them too.